src/Navbar.jsx

```diff
@@ -82,4 +90,5 @@ const Navbar = ({ sessionToken, deleteSessionToken }) => {
   );
 };
 
-export default Navbar;
+export default withCookies(Navbar);
+export { Navbar as WrappedNavbar };
```

Comment:
Do you need to export this as `WrapperNavBar`? Why not just export the component at line 31